adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostSingleton.m

```diff
@@ -203,6 +208,26 @@ - (void)presentInterstitialAdForDelegate:
   [Chartboost showInterstitial:[adapterDelegate getAdLocation]];
 }
 
+#pragma mark - Banner methods
+
+- (void)configureBannerWithSize:(GADAdSize)adSize
```

Comment:
Can we remove the Banner implementation out of the Singleton class and put it in GADMAdapterChartboost?

Unlike the Interstitial and Rewarded implementation, Banners doesn't seem to follow a singleton pattern.

Reply:
That's a good point. I moved all the management of banners to a new class GADMAdapterChartboostBanner.